lib/broccoli/ember-app.js

```diff
@@ -335,7 +335,7 @@ EmberApp.prototype._initVendorFiles = function() {
   }
 
   // don't crash if ember-cli-shims is not included
-  if (!bowerDeps['ember-cli-shims']) {
+  if (!ember && !bowerDeps['ember-cli-shims']) {
```

Comment:
This change seems somewhat odd. The idea here was that a different addon could provide the shims. That would work even with using Ember from bower.